src/main/flight/pid.c

```diff
@@ -212,7 +212,7 @@ void resetPidProfile(pidProfile_t *pidProfile)
         .idle_p = 50,
         .idle_pid_limit = 200,
         .idle_max_increase = 150,
-        .ff_interpolate_sp = FF_INTERPOLATE_ON,
+        .ff_interpolate_sp = FF_INTERPOLATE_AVG2,
```

Comment:
What are thoughts from others on this change of default?

Reply:
Other FF settings have defaults that compliment this, with it ON, there is 0 delay but it's also extremely rough, as any RC inconsistency is reflected in the motor trace, with AVG2 most of it is smoothed out, and the experience is like it was previously. With avg3 it's smoother and so on..